packages/backend/src/tokens/request.ts

```diff
@@ -48,7 +48,7 @@ function isRequestEligibleForHandshake(authenticateContext: { secFetchDest?: str
   const { accept, secFetchDest } = authenticateContext;
 
   // NOTE: we could also check sec-fetch-mode === navigate here, but according to the spec, sec-fetch-dest: document should indicate that the request is the data of a user navigation.
-  if (secFetchDest === 'document') {
+  if (secFetchDest === 'document' || secFetchDest === 'iframe') {
```

Comment:
ℹ️ When `document` requests happend from inside an iframe the `Sec-Fetch-Dest` value is set to `iframe`. Fetch/XHR requests do not use this value.

Reply:
I propose exporting a `SAFE_SEC_FETCH_DESTS` array from shared and then consuming it in backend and nextjs, and just checking `SAFE_SEC_FETCH_DESTS.includes(secFetchDest)`